src/index.js

```diff
@@ -201,28 +142,30 @@ class Provider extends React.PureComponent<Props, State> {
     return locale.split('-')[0];
   }
 
-  async getLocaleData(locale: string): Promise<Array<Object>> {
-    const language = this.localeToLanguage(locale);
-
-    const module = await import(`react-intl/locale-data/${language}`);
-    const localeData = module.default;
-
-    if (locale === 'tlh-KL') {
-      localeData.push({
-        locale: 'tlh',
-        parentLocale: 'en',
-      });
+  async setLocalData(locale: string): Promise<Array<Object>> {
```

Comment:
Minor typo here `setLocaleData` instead of `setLocalData` 🙈

Reply:
😡

Reply:
Ah yeah, lol, yesterday you actually said you made that typo 😂

Reply:
hence my angry face 😄 it's fixed now though, but approves don't appear to get reset so 👍